src/DotNetDelice/Output.fs

```diff
@@ -9,15 +9,22 @@ open Newtonsoft.Json.Serialization
 open System.IO
 open Spdx
 
+[<JsonObjectAttribute(NamingStrategyType = typeof<CamelCaseNamingStrategy>)>]
+type Package(name: string, version: string, url: string) =
```

Comment:
Can you please use a Record Type not a class here, there shouldn't be any need for a class.

Reply:
i changed class to record type. :)